modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js

```diff
@@ -54,6 +54,10 @@ const RichText = ({
 						onChange({}, data);
 					}
 				}}
+				onInstanceReady={({editor}) => {
+					editor.setData(currentValue);
+					setCurrentValue(currentValue);
```

Comment:
Why do you need to `setCurrentValue` here? Is there a chance it has changed?

Reply:
Yes, there is. When we change the form locale, we need to set the value regard to that locale